extensions/datastores/filesystem/src/main/java/org/locationtech/geowave/datastore/filesystem/config/FileSystemOptions.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2013-2020 Contributors to the Eclipse Foundation
+ *
+ * <p> See the NOTICE file distributed with this work for additional information regarding copyright
+ * ownership. All rights reserved. This program and the accompanying materials are made available
+ * under the terms of the Apache License, Version 2.0 which accompanies this distribution and is
+ * available at http://www.apache.org/licenses/LICENSE-2.0.txt
+ */
+package org.locationtech.geowave.datastore.filesystem.config;
+
+import java.nio.file.Paths;
+import java.util.Properties;
+import org.locationtech.geowave.core.store.BaseDataStoreOptions;
+import org.locationtech.geowave.core.store.DataStoreOptions;
+import org.locationtech.geowave.core.store.StoreFactoryFamilySpi;
+import org.locationtech.geowave.core.store.StoreFactoryOptions;
+import org.locationtech.geowave.datastore.filesystem.FileSystemStoreFactoryFamily;
+import org.locationtech.geowave.datastore.filesystem.util.FileSystemUtils;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.ParametersDelegate;
+
+public class FileSystemOptions extends StoreFactoryOptions {
+  public static final String DEFAULT_BINARY_FORMATTER = "binary";
+  @Parameter(
+      names = "--dir",
+      description = "The directory to read/write to.  Defaults to \"geowave\" in the working directory.")
+  private String dir = "geowave";
+
+  @Parameter(
+      names = "--format",
+      description = "Optionally use a formatter configured with Java SPI of type org.locationtech.geowave.datastore.filesystem.FileSystemDataFormatterSpi.  Defaults to \""
```

Comment:
As with the comment in the documentation, it's probably best to leave implementation details out of the CLI help.